### ISSUE TITLE
Individual membership with varying levels of benefits

### ISSUE THREAD

**Issue by ariddell** [pybee/paying-the-piper#48]
Taking a page from very well-established non-profit entities, art museums, why not allow individuals to become "members," where the benefits of membership vary depending on the contribution. It seems like one could do this not just for corporate sponsors (I gather this is already done) but also for individuals. That is, if it works for museums (and public radio in the USA) it might work for open source.

Here's one example, the Museum of Fine Arts in Boston: http://www.mfa.org/membership/levels-and-benefits . The MFA has 11 basic levels:
- Supporter $75–$109 (free parking, etc)
- Contributor  $110–$249 (extra guest passes)
- Ambassador $250–$749 (guest passes to other museums)
- Sustainer $750–$1,499 (lots of guest passes)
- Leader $1,500–$2,999 (invitation to openings of exhibits)
- Patron Member $3,000–$5,999
- Patron Fellow $6,000–$11,999
- Patron Sponsor $12,000–$29,999 
- Director's Circle $30,000–$49,999 (includes invitation to holiday party)
- President's Circle $50,000–$99,999 (dinner with director of the museum)
- Chairman's Circle Patron $100,000 or more

(there's a separate pricing tier with 4 more levels (same benefits, different titles) for younger people (who would tend to have access to less $) -- so there are even a few more levels in addition to these)

Who knows, maybe this could work? Has it been tried? I think many people would sign up to be a Django supporter if they got, say, a t-shirt and it was clear where the money was being spent (again, in keeping with the ideas in #43)

**Comment by ariddell** [comment]
Well, there is one example I know about
- Friends of GNOME https://www.gnome.org/friends/ (4 levels)

**Comment by freakboy3742** [comment]
Ok - so what can an open source project "sell" as a benefit for contribution to an open source project? A couple of quick ideas:
- Access to private events with the project maintainers
- Access to pre-sale tickets at conferences

Any other suggestions?

**Comment by ariddell** [comment]
I do wonder about the overhead in running this. If there were a lot of donors and a lot of premiums I could imagine there being a lot of administrative work. There are some nice third party systems for t-shirt rewards though (you just send the company a token and you get a t-shirt in the mail). I wonder how much this would cost for someone else to manage.

**Comment by freakboy3742** [comment]
The t-shirt idea is something that is available SaaS style form companies like [Printfection](https://printfection.com). The emergence of shirts and other merch as a backer reward on Kickstarter also means there's a bunch of companies that exist to handle this sort of order fulfilment for you.

**Comment by nayafia** [comment]
I like this idea but I'm a little stuck on ideas for tangible benefits to offer members vs. museums, etc.

Major donors to the arts give because they want access + status symbol. It's sort of an "in circle" to mingle with other high net worth patrons and a way to show your status. Would those mechanics work here? Maybe something like Django Under the Hood for patrons?

**Comment by freakboy3742** [comment]
@nayafia A completely separate conference probably wouldn't work, because organising a conference (especially one with invited speakers like DuTH) is expensive, both in terms of actual costs, and opportunity cost - a week at a conference is a week away from work. 

However, _availability_ of tickets for DuTH is a big issue - they sell out in about 5 minutes. Presale access for sponsors, or a guaranteed pool of tickets for sponsors at a particular level - _that_ might be workable.

Another idea that has been floated is "season pass" to DjangoCons - at a high enough sponsorship level, a company gets a guaranteed ticket for every DjangoCon event. Some of that money goes to the conference like a regular sponsorship.

**Comment by lovelydinosaur** [comment]
Using reserved conf tickets as fringe benefit of higher sponsorship tiers is pretty smart. Nice simple easy sell, that.

**Comment by tleeuwenburg** [comment]
Things I would pay for:
  -- Access to exclusive events with influential developers and leaders
  -- Access to prioritisation of feature requests and bugs
  -- Having my name listed on a list of financial members

Not much else. This would genuinely be something I would do for professional rather than personal or social reasons, so I have concentrated on things I think make sense in the professional context.

**Comment by nanuxbe** [comment]
@tleeuwenburg this sounds to me very much like finding ways to get already privileged people more privileges. (If you have money to give, you'll be able to meet influential people).

While this is probably a very profitable angle which has brought a lot of money to other orgs worldwide I have a strong feeling that this goes against the newly found "open, inclusive and diverse" policies of Open Source.

Also, even if you are willing to pay for this, this is once again going to get money out of the pocket of developer (some of them already contributing back to OS in various ways) instead of getting it out of the corps pockets.

**Comment by tleeuwenburg** [comment]
@nanuxbe yes and no. I think this is a nuanced point. Got to go now but I think I need to provide some more balanced comments on this at risk of being misunderstood

**Comment by nayafia** [comment]
@nanuxbe Yeah, that's why I hesitated on being able to name any tangible benefits that could be offered for memberships. Because the way it works for, say, high-end patrons of an art museum IS all about increasing access + privilege for people who are able to pay. If you don't make it about status symbols, those memberships look more like crowdfunding (get a T-shirt, etc) and that model already has its issues.

**Comment by tleeuwenburg** [comment]
So where I wanted to go was ... it's hard to keep the money out of it, without keeping money out of it. I think it goes to motivation. If you're "selling" membership in terms of direct benefits to members, then of course commercial interests will come into it. If you're simply "charging" for membership, then it can be seen as more of a way for people to provide support back to their industry.

In terms of your comment on privilege, I think it's hard to talk about a commercial arrangement based on the provision of value which doesn't inherently come with that. 

I think, though, it's possible to create 'arms length' arrangements, whereby the integrity of the association can be protected through independence. If the professional association has one activity which is sponsoring project work, that should be separated from the commercial activities of that association.

This conversation is all around, I think, that general conversation.

If the goal is to get corporations to pay, rather than to raise money from professional individuals, then realistically you need to enter the corporate market and sell them something. Creating scarcity through licensing is possible, but you're still not going to be able to simply build and sell software -- there still needs to be some kind of business model around "solving problems" rather than selling software.

Is there a clear answer to "where should the money come from"?

Thanks for the interesting conversation. I hope everyone understands that I am coming from the perspective of "exploring ideas" rather than claiming correctness...

**Comment by ariddell** [comment]
I happened to be in a museum today and had one more thought about this.

@nayafia @nanuxbe If the amount given/donated dwarfs the "reward" I think the situation is less problematic. If someone gives $100,000 to the Django foundation -- they're not buying access; they're donating. Having dinner with them is a completely appropriate gesture of gratitude.

Another analogy might be with a large donation to a university. If someone donates $100 million for a new building, putting their name somewhere on/in the building seems unobjectionable. Naturally you have to make very clear that there's no influence being purchased but there are plenty of examples that show this is possible.

**Comment by wolftune** [comment]
@ariddell while giving everyone the same access and only doin very minor special things for donors is _far, far_ better than reserving exclusive access to actually valuable things, all large grants still mean that the project will feel the need to appeal to the interests of large donors, not turn them away, etc. — it's not ideal to rely on large donors, and it still shifts the influences and project decisions toward the interests of the privileged. Not saying that trade-off isn't a necessary evil, but it needs to be acknowledged.

**Comment by lovelydinosaur** [comment]
I would stop talking in terms of 'donate' and start talking in terms of 'invest'. The most valuable thing that companies that invest in the OSS projects gain is time being spent on the project. Other rewards are useful, but they are fringe benefits rather than the primary reward.

**Comment by ariddell** [comment]
I think we're thinking about different kinds of open source projects.
Perhaps this donation-focused approach would be more appropriate for
projects which tend not to focus on providing tangible benefits to
for-profit corporations.

**Comment by lovelydinosaur** [comment]
Possibly so
